Formula/skaffold.rb

```diff
@@ -2,8 +2,8 @@ class Skaffold < Formula
   desc "Easy and Repeatable Kubernetes Development"
   homepage "https://github.com/GoogleContainerTools/skaffold"
   url "https://github.com/GoogleContainerTools/skaffold.git",
-      :tag      => "v1.4.0",
-      :revision => "bb324609b4254ceb9f76d35eb03642a6a1807867"
+      :tag      => "1.5.0",
```

Comment:
Looks like the tag is actually `v1.5.0`.

```suggestion
      :tag      => "v1.5.0",
```